src/app/address/fields.js

```diff
@@ -22,5 +22,18 @@ module.exports = {
   "address-search": {
     type: "text",
     autocomplete: "Postcode",
+    validate: [
```

Comment:
As discussed offline, we'll start off by adding minimal validation and increasing it as we decide we need to - unlike many other systems, we are checking this via an API afterwards, and that will catch any postcodes that aren't matched.